Create loc-only-mt-engine-updates.md (#17294)
dotnet/docs: CONTRIBUTING.md

- [.NET Compiler Platform SDK reference](https://github.com/dotnet/roslyn-api-docs)
    Issues and tasks for this repo are tracked in [dotnet/docs/issues](https://github.com/dotnet/docs/issues).

### Contributing to International content

> Contribution for Machine Translated (MT) content are currently not accepted and deactivated for now. In an effort to improve the quality of MT content, we've transitioned to a Neural MT engine. We're focusing our feedback efforts on human translated content, which is used to train the Neural MT engine. So over time, the contributions will improve the quality of both human and machine translation. MT topics will have a disclaimer stating that part of the topic may be machine translated, and the **Edit** button won't be displayed as its disabled. 

Reviewer: ```suggestion
Contributions for Machine Translated (MT) content are currently not accepted. In an effort to improve the quality of MT content, we've transitioned to a Neural MT engine. We're focusing our feedback efforts on human translated content, which is used to train the Neural MT engine. So over time, the contributions will improve the quality of both human and machine translation. MT topics will have a disclaimer stating that part of the topic may be machine translated, and the **Edit** button won't be displayed because editing is disabled. 
```

- [.NET Compiler Platform SDK reference](https://github.com/dotnet/roslyn-api-docs)
    Issues and tasks for this repo are tracked in [dotnet/docs/issues](https://github.com/dotnet/docs/issues).

> [!INCLUDE [](includes/loc-only-mt-engine-updates.md)]

Reviewer 2: The contributing file is a repo only file that doesn't accept Markdown extensions. So this would show up for customers reading this.

Author: Do you mean that INCLUDES reference won't work for CONTRIBUTING.md, so I should directly add the content?

Reviewer 2: Yep. This what customers will see right now: https://github.com/dotnet/docs/blob/3b24abde90b95f531799197b392667706b0aa7c3/CONTRIBUTING.md

Author: I see, thank you. I've replaced the INCLUDES reference with proposed content and deleted the INCLUDES file from this PR.
